executor/executor.cc

```diff
@@ -1543,4 +1544,4 @@ void debug_dump_data(const char* data, int length)
 	}
 	if (i % 16 != 0)
 		debug("\n");
-}
+}
```

Comment:
Please undo unnecessary newline change.

Reply:
@Merna177 please fix (there was a newline following this brace)